apps/sms/views/conversation/js/conversation.js

```diff
@@ -769,10 +779,11 @@ var ConversationView = {
         return;
       }
 
+      var promises = [];
       // Recipients will exist for draft messages in threads
       // Otherwise find them from draft recipient numbers
       this.draft.recipients.forEach(function(number) {
-        Contacts.findByAddress(number).then(function(contacts) {
+        promises.push(Contacts.findByAddress(number).then(function(contacts) {
```

Comment:
this `then` callback was executed after `draftPromise.then` line 648. In other word, now that we check the recipients to decide where to focus, we need to wait for their initialization